idea_town/experiments/migrations/0003_auto_20151021_2102.py

```diff
@@ -49,4 +49,14 @@ class Migration(migrations.Migration):
             name='measurements_markup_type',
             field=models.CharField(choices=[('', '--'), ('html', 'HTML'), ('plain', 'Plain'), ('markdown', 'Markdown'), ('restructuredtext', 'Restructured Text')], max_length=30, default='plain', blank=True),
         ),
+        migrations.AddField(
```

Comment:
I think this is squashed *too* far back. This migration already exists with a previous commit, we need a new migration for this new set of changes.